go-controller/pkg/ovn/controller/services/lb_config.go

```diff
@@ -41,45 +43,42 @@ type lbConfig struct {
 	hasNodePort bool
 }
 
-func (c *lbConfig) makeNodeSwitchTargetIPs(node *nodeInfo, epIPs []string) (targetIPs []string, changed bool) {
-	targetIPs = epIPs
-	changed = false
+func (c *lbConfig) makeNodeSwitchTargetIPs(service *v1.Service, node *nodeInfo, clusterEndpoints []string, localEndpoints []string) (
```

Comment:
Also no need for `node`

Actually cluster endpoints and local endpoints are also part of lbconfig so it does look weird.

You could frame this as a free method
`func getNodeSwitchTargetV4IPs(node string, c *lbConfig) (targetIPs []string, isLocal bool)`

Reply:
Good idea! I made `getNodeSwitchTargetIPs` handle v4 and v6 at the same time. Does it look ok?
Also, do you prefer a free method because we're not really modifying `c *lbConfig`?